src/resources/views/admin/editQuestions.blade.php

```diff
@@ -0,0 +1,25 @@
+@extends('layouts.kuizy')
+
+@section('title')
+@foreach($big_items as $big_item)
+<h1> {{ $big_item -> name }}</h1>
+@section('menubar')
+<a href="/kuizy/admin/add">タイトル一覧へ戻る</a>
+@endsection
+
+@section('question')
+
+<form action="/{{ request()->path() }}" method="POST" enctype="multipart/form-data">
+    <ul class="choices">
+        @foreach($big_item->questions as $item)
```

Comment:
```suggestion
        @foreach($big_item->questions as $question_item)
```
の方が今までのやり方に似てる気がします！